README.md

```diff
@@ -28,11 +28,13 @@ Setup for developers
   shell variable every time you activate the virtualenv, edit `venv/bin/activate`
   and add to the bottom the export statement.
 7. Run `python systers_portal/manage.py migrate`.
-8. Run `python systers_portal/manage.py runserver` to start the development server. When in testing
+8. Run `python systers_portal/manage.py createsuperuser` to create a superuser for the admin pannel.
+   Fill in the details asked.
```

Comment:
Word "pannel" should have one "n". And 2 spaces are enough (you have 3) - just for the consistency.